node/node.go

```diff
@@ -151,7 +152,11 @@ func (node *Node) Blockchain() *core.BlockChain {
 
 // Beaconchain returns the beaconchain from node.
 func (node *Node) Beaconchain() *core.BlockChain {
-	bc, err := node.shardChains.ShardChain(shard.BeaconChainShardID)
+	return node.EpochChain()
```

Comment:
Please don't change the existing Beaconchain() which is used somewhere else which may require the full chain. Beaconchain should mean the full chain while EpochChain means the chain with only epoch blocks. Beaconchain can be used as a EpochChain, but EpochChain can't be used as Beaconchain.

Reply:
Fixed